roles/deploy_olm_operator/tasks/main.yml

```diff
@@ -207,6 +207,19 @@
       set_fact:
         deploy_olm_operator_results: {'result': 'pass', 'msg': ''}
 
+    - name: "Get replicaset logs for operators that failed olm-deployment output"
+      shell: "{{ oc_bin_path }} get replicasets -o go-template={%raw%}'{{range .items}}{{.status.conditions}}{{\"\\n\"}}{{end}}'{%endraw%} -n {{ openshift_namespace }}"
```

Comment:
Don't some operators deploy to multiple namespaces?  Should we be using --all-namespaces?

Reply:
Operators can target and manage multiple namespaces, but are installed in a single namespace - the `openshift_namespace` in our case.

Reply:
Got it